satpy/readers/clavrx.py

```diff
@@ -94,9 +108,29 @@ def _get_rows_per_scan(sensor: str) -> Optional[int]:
     return None
 
 
+def _scale_data(data_arr: Union[xr.DataArray, int], scale_factor: float, add_offset: float) -> xr.DataArray:
```

Comment:
I'm not requesting a change here, but FYI if you do `from __future__ import annotations` at the top of the module you can then use `xr.DataArray | int` here instead of the `Union`.